compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/expressions/InvocationNode.java

```diff
@@ -34,7 +34,7 @@
 
     List<? extends ExpressionNode> getArgumentExpressions();
 
-    VariableReferenceNode getExpression();
+    ExpressionNode getExpression();
```

Comment:
I assume this is done to support invocations done on top of literals right? like
string value = "abc".concat("cde");

Reply:
Yes, this is done to support the mentioned scenario.